lnrpc/rpc.proto

```diff
@@ -203,6 +203,8 @@ message SendRequest {
     string payment_hash_string = 5;
 
     string payment_request = 6;
+
+    repeated Route routes = 7 [ json_name = "routes"];
```

Comment:
Rather than just tacking on a `routes` field to this RPC, an entirely new RPC should be added as detailed in the original, issue.